lib/signet/transaction.ex

```diff
@@ -417,13 +422,15 @@ defmodule Signet.Transaction do
           max_priority_fee_per_gas,
           max_fee_per_gas,
           gas_limit,
-          destination,
+          String.trim_leading(destination),
```

Comment:
`trim_leading/1`? what does that do?

Reply:
This was necessary to match expectations.